### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by bputzeys**

### CODE REVIEW

File: helical/models/scgpt/LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2023 Yanay Rosen, Yusuf Roohani, Jure Leskovec
```
Comment (bputzeys) [inline helical/models/scgpt/LICENSE:3]: @maxiallard is this intended?

File: requirements.txt [hunk 2]
```diff
@@ -3,13 +3,15 @@ pandas==2.2.2
...
 scikit-learn==1.2.2
+scipy==1.13.1
 gitpython==3.1.43
 torch==2.3.0
 torchvision==0.18.0
 accelerate==0.29.3
 transformers==4.26.1
 loompy==3.0.7
 scib==1.1.5
+scikit-misc==0.3.1
```
Comment (bputzeys) [inline requirements.txt:14]: https://github.com/has2k1/scikit-misc/issues/37

File: requirements.txt [hunk 3]
```diff
@@ -3,13 +3,15 @@ pandas==2.2.2
 anndata==0.10.7
 numpy==1.26.4
 scikit-learn==1.2.2
+scipy==1.13.1
```
Comment (bputzeys) [inline requirements.txt:6]: Version 1.14.0 caused the following error:
```
    data = scgpt.process_data(adata[:10])
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/bputzeys/Documents/Helical/Code_Repos/helical/helical/models/scgpt/model.py", line 203, in process_data
    count_matrix = (adata.X if isinstance(adata.X, np.ndarray) else adata.X.A)
                                                                    ^^^^^^^^^
AttributeError: 'SparseCSRView' object has no attribute 'A'
```

File: helical/models/scgpt/LICENSE (hunk 1 above)
Comment (maxiallard) [reply helical/models/scgpt/LICENSE:3]: yes

**Review by maxiallard — APPROVED** [review]

**Pull request merged**